Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/GestureSettings.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Definitions.InputSystem
+{
+    /// <summary>
+    /// Copy of Unity's XR.WSA.GestureSettings.
+    /// </summary>
+    [Flags]
+    public enum GestureSettings
```

Comment:
Rename to WindowsMixedRealityGestureSettings? Or do we expect to add additional platform gestures here in the future?

Reply:
Needs to be cross platform. This isn't platform specific so I won't be renaming it.

Reply:
I can see us adding to this in the future as more platforms get supported.